structs.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+type TestMapping struct {
+	Name     string   `json:"name,omitempty"`
+	Strategy string   `json:"strategy,omitempty"`
+	Values   []string `json:"values,omitempty"`
+}
+
+type TestSharding struct {
+	NumberOfShards      int           `json:"numberOfShards,omitempty"`
+	Mapping             []TestMapping `json:"mapping,omitempty"`
+	AutoStrategyDevices []string      `json:"devices,omitempty"`
+}
+
+type TestCases struct {
+	Count  int                    `json:"count,omitempty"`
+	Status map[string]interface{} `json:"status,omitempty"`
+}
+
+type Sessions struct {
+	Id        string `json:"id"`
+	Status    string `json:"status"`
+	TestCases string `json:"start_time"`
+}
+
+type BrowserStackPayload struct {
+	App                    string      `json:"app"`
+	TestSuite              string      `json:"testSuite"`
+	Devices                []string    `json:"devices"`
+	InstrumentationLogs    bool        `json:"instrumentationLogs"`
+	NetworkLogs            bool        `json:"networkLogs"`
+	DeviceLogs             bool        `json:"deviceLogs"`
+	DebugScreenshots       bool        `json:"debugscreenshots,omitempty"`
+	VideoRecording         bool        `json:"video"`
+	Project                string      `json:"project,omitempty"`
+	ProjectNotifyURL       string      `json:"projectNotifyURL,omitempty"`
+	UseLocal               bool        `json:"local,omitempty"`
+	ClearAppData           bool        `json:"clearPackageData,omitempty"`
+	SingleRunnerInvocation bool        `json:"singleRunnerInvocation,omitempty"`
+	Class                  []string    `json:"class,omitempty"`
+	Package                []string    `json:"package,omitempty"`
+	Annotation             []string    `json:"annotation,omitempty"`
+	Size                   []string    `json:"size,omitempty"`
+	UseMockServer          bool        `json:"allowDeviceMockServer,omitempty"`
+	UseTestSharding        interface{} `json:"shards,omitempty"`
+
+	// non ui fields
```

Comment:
why are these commented?

Reply:
As discussed with @deepwired over the call, added a better comment which explains why we've commented params.